### PR TITLE
AKS Baseline Multi Cluster - Adding a note to inform the user about the deprecated --sdk-auth option when creating the github workflow's credentials 

### PR DESCRIPTION
**Pull request by magrande**

### CODE REVIEW

File: docs/deploy/06-aks-cluster.md [hunk 1]
```diff
@@ -42,6 +42,13 @@ Following the steps below will result in the provisioning of the AKS multi clust
 
         ```bash
         # Create an Azure Service Principal
+        #
+        # Note that you will be running the next command using the --sdk-auth flag which is marked 
+        # as deprecated; however, the github workflow's Azure login action, using the credentials 
+        # generated by this command's output will still be successful. The Azure login action 
+        # supports an improved way of authenticating with Azure using a Federated Identity 
+        # Credential. We plan on including this new method in the next updates.
+        #
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:51]: How about shortening the message to these three lines?

```suggestion
        #
        # This command will generate a sp.json file that in the format expected by the GitHub Actions for
        # Azure. This is accomplished by using the deprecated --sdk-auth flag. For alternatives, including
        # using Federated Identity, see: https://github.com/Azure/login#configure-deployment-credentials
```

**Review by ckittel — CHANGES_REQUESTED** [review]
In addition to considering the wording update I provided, can we move to `v1` of the login action?  That'll make sure that federated identity is actually possible (as it isn't in `v1.1`)

File: docs/deploy/06-aks-cluster.md [hunk 2]
```diff
@@ -42,6 +42,10 @@ Following the steps below will result in the provisioning of the AKS multi clust
 
         ```bash
         # Create an Azure Service Principal
+        #
+        # This command will generate a sp.json file that in the format expected by the GitHub Actions for
+        # Azure. This is accomplished by using the deprecated --sdk-auth flag. For alternatives, including
+        # using Federated Identity, see: https://github.com/Azure/login#configure-deployment-credentials
```
Comment (ckittel) [inline docs/deploy/06-aks-cluster.md:48]: Sorry, I had a typo in there!

```suggestion
        #
        # This command will generate a sp.json file in the format expected by GitHub Actions for Azure.
        # This is accomplished by using the deprecated --sdk-auth flag. For alternatives, including using
        # Federated Identity, see https://github.com/Azure/login#configure-deployment-credentials.
```

**Comment by magrande** [comment]
> In addition to considering the wording update I provided, can we move to `v1` of the login action? That'll make sure that federated identity is actually possible (as it isn't in `v1.1`)

Makes sense, done !

**Review by ckittel — APPROVED** [review]
:shipit:

**Pull request merged**